openmetadata-ui/src/main/resources/ui/src/pages/AlertDetailsPage/AlertDetailsPage.tsx

```diff
@@ -318,145 +319,171 @@ function AlertDetailsPage({
-        className: 'content-resizable-panel-container',
-      }}
-    />
+                        {deletePermission &&
+                          alertDetails?.provider !== ProviderType.System && (
+                            <Tooltip
+                              title={t('label.delete-entity', {
+                                entity: t('label.alert'),
+                              })}>
+                              <Button
+                                className="flex flex-center"
+                                data-testid="delete-button"
+                                icon={<DeleteIcon height={16} width={16} />}
+                                onClick={() => setShowDeleteModal(true)}
+                              />
+                            </Tooltip>
+                          )}
+                      </Space>
+                    </Col>
+                  </Row>
+                </Col>
+
+                <Col
+                  className="alert-description"
+                  data-testid="alert-description"
+                  span={24}>
+                  <DescriptionV1
+                    description={alertDetails?.description}
+                    entityType={EntityType.EVENT_SUBSCRIPTION}
+                    hasEditAccess={editDescriptionPermission}
+                    isEdit={showDescriptionModal}
+                    showCommentsIcon={false}
+                    onCancel={onCancel}
+                    onDescriptionEdit={onDescriptionEdit}
+                    onDescriptionUpdate={onDescriptionUpdate}
+                  />
+                </Col>
+
+                <Col span={24}>
+                  <Tabs
+                    activeKey={tab}
+                    className="m-b-lg"
+                    items={tabItems}
+                    onTabClick={handleTabChange}
+                  />
+                </Col>
+              </Row>
+              <DeleteWidgetModal
+                afterDeleteAction={handleAlertDelete}
+                allowSoftDelete={false}
+                entityId={alertDetails?.id ?? ''}
+                entityName={getEntityName(alertDetails)}
+                entityType={EntityType.SUBSCRIPTION}
+                visible={showDeleteModal}
+                onCancel={hideDeleteModal}
+              />
+            </div>
+          ),
+          minWidth: 700,
+          flex: 0.7,
+        }}
+        pageTitle={t('label.entity-detail-plural', {
+          entity: t('label.alert'),
+        })}
+        secondPanel={{
+          children: <></>,
+          minWidth: 0,
+          className: 'content-resizable-panel-container',
+        }}
+      />
+    );
+  }, [
+    loadingCount,
+    viewPermission,
+    alertDetails,
+    breadcrumb,
+    alertIcon,
+    ownerLoading,
+    editOwnersPermission,
+    extraInfo,
+    editPermission,
+    deletePermission,
+    showDeleteModal,
+    tab,
+    tabItems,
+    editDescriptionPermission,
+    showDescriptionModal,
+  ]);
```

Comment:
Let's make sure we have included all the variables in the dependency array here as well